### PR TITLE
add streaming model explorer demo

### PR DESCRIPTION
**Pull request by annthurium**
https://github.com/deepset-ai/devrel-board/issues/287

here is a version of the notebook in Google Drive in case you want to test it there: https://colab.research.google.com/drive/1aVp_f9MUAIR6eczASSOpsbMopjg00wyj#scrollTo=T8zE8SbZ0wUz

### CODE REVIEW

File: notebooks/model_explorer_streaming.ipynb [hunk 1]
```diff
@@ -0,0 +1,1140 @@
+{
```
Comment (TuanaCelik) [inline notebooks/model_explorer_streaming.ipynb:1]: <p>If you like, you could add a line at the top here that says: _by Tilde Thurium: LI &amp; Twitter_ or any social you'd like people to follow you on :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (TuanaCelik) [inline notebooks/model_explorer_streaming.ipynb:1]: <p>How are these userdate.get lines supposed to work? Trying this with no prior step in the colab fails, maybe we should have a cell where we ask for user input? Or instructions on how to set the userdata? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (annthurium) [reply notebooks/model_explorer_streaming.ipynb:1]: Oh actually there's documentation about that in the first text block. I can move that down here to make it more visible.

**Review by TuanaCelik — APPROVED** [review]
Merge away!

**Pull request merged**